### PR TITLE
Added empty state token

### PR DESCRIPTION
**Pull request by Pallavi-PH**

### CODE REVIEW

**Comment by visualapps** [comment]
We need to have more variations for blank-slate component

With illustration
Without illustration
With/Without call-to-action buttons

File: docs/tokens/emptystate.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+menu: Tokens
+
+---
+import { Playground, PropsTable } from 'docz'
+import emptyState from '../../resources/images/illustrations/error-404.svg'
+
+# Empty State
+
+<Playground>
+    <div class="card_zero_result">
```
Comment (visualapps) [inline docs/tokens/emptystate.mdx:12]: class name should follow the BEM conventions.

examples

```
<div class="blank-slate">
   <div class="blank-slate-content">
      <div class="blank-slate-content__illustration">
      </div>
      <div class="blank-slate-content__message"></div>
      <div class="blank-slate-content__cta"></div>
   </div>
</div>
</div>
```

**Comment by visualapps** [comment]
https://www.carbondesignsystem.com/patterns/empty-states-pattern/

File: docs/tokens/emptystate.mdx (hunk 1 above)
Comment (visualapps) [reply docs/tokens/emptystate.mdx:12]: https://success.outsystems.com/Documentation/11/Developing_an_Application/Design_UI/Patterns/Using_Traditional_Web_Patterns/Content/BlankSlate

Comment (visualapps) [reply docs/tokens/emptystate.mdx:12]: https://styleguide.github.com/primer/components/blankslate/

File: scss/core/tokens/_emptystate.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.blank-slate {
+    border-radius: 4px;
+    padding: 46px 0;
+
+    .blank-slate-content{
+      text-align: center;
+
+      .blank-slate-content__message {
+        color: $gray-700;
+        font-size: 1.1rem;
```
Comment (visualapps) [inline scss/core/tokens/_emptystate.scss:10]: font-size can we have in px

File: scss/_core.scss [hunk 3]
```diff
@@ -17,3 +17,4 @@
 
 //Utiliticore/es
 @import "core/tokens/_sizing.scss";
+@import "core/tokens/emptystate"
```
Comment (visualapps) [inline scss/_core.scss:20]: Empty-state do we want to keep in tokens or blocks?

Comment (visualapps) [reply scss/_core.scss:20]: it doesn't seems like a token, it's a component itself

Comment (Pallavi-PH) [reply scss/_core.scss:20]: Will move it to blocks

**Comment by Pallavi-PH** [comment]
Closing this PR. Included these changes and updated them with the latest UX in this PR https://github.com/mayadata-io/de-code/pull/264

**Pull request closed without merging**